empty-resources/0000_05_config-operator_02_operatorhub.cr.yaml

```diff
@@ -7,4 +7,5 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
     release.openshift.io/create-only: "true"
+    capability.openshift.io/name: "marketplace"
```

Comment:
Should I actually be scrambling to move this to `/manifests` in `operator-marketplace`?